lib/concurrent/atomic/thread_local_var.rb

```diff
@@ -1,3 +1,5 @@
+require 'concurrent/atomic/weak_key_map'
```

Comment:
I'll put it in a subdir. I didn't want to embed it directly in the class because it's a lot of code that isn't always activated. When it's in the same file as `AbstractThreadLocalVar` it makes the code for that class much harder to read.

I chose to nest it under `AbstractThreadLocalVar` because it's only needed on a few runtimes. I didn't want to make it a top-level, public class.